### ISSUE TITLE
Building wheel for pyjls on RPi fails

### ISSUE THREAD

**Issue by odurc** [jetperch/jls#8]
Run the steps below on a Raspberry Pi 4 to reproduce.

```
pi@rpi[+1]/tmp ➜ virtualenv .venv
created virtual environment CPython3.9.2.final.0-32 in 1139ms
  creator CPython3Posix(dest=/tmp/.venv, clear=False, no_vcs_ignore=False, global=False)
  seeder FromAppData(download=False, pip=bundle, setuptools=bundle, wheel=bundle, via=copy, app_data_dir=/home/pi/.local/share/virtualenv)
    added seed packages: pip==23.1.2, setuptools==67.7.2, wheel==0.40.0
  activators BashActivator,CShellActivator,FishActivator,NushellActivator,PowerShellActivator,PythonActivator
pi@rpi[+1]/tmp ➜ source .venv/bin/activate
pi@rpi[+1].venv|/tmp ➜ pip install joulescope
Looking in indexes: https://pypi.org/simple, https://www.piwheels.org/simple
Collecting joulescope
  Using cached https://www.piwheels.org/simple/joulescope/joulescope-1.1.7-cp39-cp39-linux_armv7l.whl (2.7 MB)
Collecting numpy (from joulescope)
  Using cached https://www.piwheels.org/simple/numpy/numpy-1.24.3-cp39-cp39-linux_armv7l.whl (12.3 MB)
Collecting psutil (from joulescope)
  Using cached https://www.piwheels.org/simple/psutil/psutil-5.9.5-cp39-abi3-linux_armv7l.whl (276 kB)
Collecting pyjls>=0.7.1 (from joulescope)
  Using cached pyjls-0.7.1.tar.gz (278 kB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... done
Collecting pyjoulescope-driver>=1.3.14 (from joulescope)
  Using cached pyjoulescope_driver-1.3.15.tar.gz (3.3 MB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... done
Collecting python-dateutil>=2.7.3 (from joulescope)
  Using cached https://www.piwheels.org/simple/python-dateutil/python_dateutil-2.8.2-py2.py3-none-any.whl (247 kB)
Collecting pymonocypher>=3.1.3.2 (from joulescope)
  Using cached https://www.piwheels.org/simple/pymonocypher/pymonocypher-3.1.3.2-cp39-cp39-linux_armv7l.whl (301 kB)
Collecting requests (from pyjoulescope-driver>=1.3.14->joulescope)
  Using cached https://www.piwheels.org/simple/requests/requests-2.31.0-py3-none-any.whl (62 kB)
Collecting six>=1.5 (from python-dateutil>=2.7.3->joulescope)
  Using cached https://www.piwheels.org/simple/six/six-1.16.0-py2.py3-none-any.whl (11 kB)
Collecting charset-normalizer<4,>=2 (from requests->pyjoulescope-driver>=1.3.14->joulescope)
  Using cached charset_normalizer-3.1.0-py3-none-any.whl (46 kB)
Collecting idna<4,>=2.5 (from requests->pyjoulescope-driver>=1.3.14->joulescope)
  Using cached https://www.piwheels.org/simple/idna/idna-3.4-py3-none-any.whl (61 kB)
Collecting urllib3<3,>=1.21.1 (from requests->pyjoulescope-driver>=1.3.14->joulescope)
  Using cached https://www.piwheels.org/simple/urllib3/urllib3-2.0.3-py3-none-any.whl (123 kB)
Collecting certifi>=2017.4.17 (from requests->pyjoulescope-driver>=1.3.14->joulescope)
  Using cached https://www.piwheels.org/simple/certifi/certifi-2023.5.7-py3-none-any.whl (156 kB)
Building wheels for collected packages: pyjls, pyjoulescope-driver
  Building wheel for pyjls (pyproject.toml) ... error
  error: subprocess-exited-with-error
  
  × Building wheel for pyjls (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [29 lines of output]
      running bdist_wheel
      running build
      running build_py
      creating build
      creating build/lib.linux-armv7l-cpython-39
      creating build/lib.linux-armv7l-cpython-39/pyjls
      copying pyjls/__main__.py -> build/lib.linux-armv7l-cpython-39/pyjls
      copying pyjls/__init__.py -> build/lib.linux-armv7l-cpython-39/pyjls
      copying pyjls/structs.py -> build/lib.linux-armv7l-cpython-39/pyjls
      copying pyjls/version.py -> build/lib.linux-armv7l-cpython-39/pyjls
      creating build/lib.linux-armv7l-cpython-39/pyjls/test
      copying pyjls/test/__init__.py -> build/lib.linux-armv7l-cpython-39/pyjls/test
      copying pyjls/test/test_binding.py -> build/lib.linux-armv7l-cpython-39/pyjls/test
      creating build/lib.linux-armv7l-cpython-39/pyjls/v1
      copying pyjls/v1/__init__.py -> build/lib.linux-armv7l-cpython-39/pyjls/v1
      creating build/lib.linux-armv7l-cpython-39/pyjls/entry_points
      copying pyjls/entry_points/plot.py -> build/lib.linux-armv7l-cpython-39/pyjls/entry_points
      copying pyjls/entry_points/info.py -> build/lib.linux-armv7l-cpython-39/pyjls/entry_points
      copying pyjls/entry_points/annotate.py -> build/lib.linux-armv7l-cpython-39/pyjls/entry_points
      copying pyjls/entry_points/__init__.py -> build/lib.linux-armv7l-cpython-39/pyjls/entry_points
      copying pyjls/entry_points/export.py -> build/lib.linux-armv7l-cpython-39/pyjls/entry_points
      running build_ext
      building 'pyjls.binding' extension
      creating build/temp.linux-armv7l-cpython-39
      creating build/temp.linux-armv7l-cpython-39/pyjls
      creating build/temp.linux-armv7l-cpython-39/src
      arm-linux-gnueabihf-gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O2 -Wall -g -ffile-prefix-map=/python3.9-3.9.2=. -fstack-protector-strong -Wformat -Werror=format-security -g -fwrapv -O2 -fPIC -Iinclude -Iinclude_prv -I/tmp/pip-build-env-zdta87f_/overlay/lib/python3.9/site-packages/numpy/core/include -I/tmp/.venv/include -I/usr/include/python3.9 -c pyjls/binding.c -o build/temp.linux-armv7l-cpython-39/pyjls/binding.o -msse4.2
      arm-linux-gnueabihf-gcc: error: unrecognized command-line option ‘-msse4.2’
      error: command '/usr/bin/arm-linux-gnueabihf-gcc' failed with exit code 1
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for pyjls
  Building wheel for pyjoulescope-driver (pyproject.toml) ... error
  error: subprocess-exited-with-error
  
  × Building wheel for pyjoulescope-driver (pyproject.toml) did not run successfully.
  │ exit code: 1
  ╰─> [301 lines of output]
      running bdist_wheel
      running build
      running build_py
      creating build
      creating build/lib.linux-armv7l-cpython-39
      creating build/lib.linux-armv7l-cpython-39/pyjoulescope_driver
      copying pyjoulescope_driver/__main__.py -> build/lib.linux-armv7l-cpython-39/pyjoulescope_driver
      copying pyjoulescope_driver/time64.py -> build/lib.linux-armv7l-cpython-39/pyjoulescope_driver
      copying pyjoulescope_driver/program.py -> build/lib.linux-armv7l-cpython-39/pyjoulescope_driver
      copying pyjoulescope_driver/__init__.py -> build/lib.linux-armv7l-cpython-39/pyjoulescope_driver
      copying pyjoulescope_driver/release.py -> build/lib.linux-armv7l-cpython-39/pyjoulescope_driver
      copying pyjoulescope_driver/version.py -> build/lib.linux-armv7l-cpython-39/pyjoulescope_driver
      copying pyjoulescope_driver/record.py -> build/lib.linux-armv7l-cpython-39/pyjoulescope_driver
      creating build/lib.linux-armv7l-cpython-39/pyjoulescope_driver/test
      copying pyjoulescope_driver/test/test_release.py -> build/lib.linux-armv7l-cpython-39/pyjoulescope_driver/test
      copying pyjoulescope_driver/test/__init__.py -> build/lib.linux-armv7l-cpython-39/pyjoulescope_driver/test
      copying pyjoulescope_driver/test/test_time64.py -> build/lib.linux-armv7l-cpython-39/pyjoulescope_driver/test
      creating build/lib.linux-armv7l-cpython-39/pyjoulescope_driver/entry_points
      copying pyjoulescope_driver/entry_points/program.py -> build/lib.linux-armv7l-cpython-39/pyjoulescope_driver/entry_points
      copying pyjoulescope_driver/entry_points/info.py -> build/lib.linux-armv7l-cpython-39/pyjoulescope_driver/entry_points
      copying pyjoulescope_driver/entry_points/__init__.py -> build/lib.linux-armv7l-cpython-39/pyjoulescope_driver/entry_points
      copying pyjoulescope_driver/entry_points/gpi.py -> build/lib.linux-armv7l-cpython-39/pyjoulescope_driver/entry_points
      copying pyjoulescope_driver/entry_points/statistics.py -> build/lib.linux-armv7l-cpython-39/pyjoulescope_driver/entry_points
      copying pyjoulescope_driver/entry_points/threads.py -> build/lib.linux-armv7l-cpython-39/pyjoulescope_driver/entry_points
      copying pyjoulescope_driver/entry_points/set_parameter.py -> build/lib.linux-armv7l-cpython-39/pyjoulescope_driver/entry_points
      copying pyjoulescope_driver/entry_points/api_timeout.py -> build/lib.linux-armv7l-cpython-39/pyjoulescope_driver/entry_points
      copying pyjoulescope_driver/entry_points/scan.py -> build/lib.linux-armv7l-cpython-39/pyjoulescope_driver/entry_points
      copying pyjoulescope_driver/entry_points/record.py -> build/lib.linux-armv7l-cpython-39/pyjoulescope_driver/entry_points
      running egg_info
      writing pyjoulescope_driver.egg-info/PKG-INFO
      writing dependency_links to pyjoulescope_driver.egg-info/dependency_links.txt
      writing entry points to pyjoulescope_driver.egg-info/entry_points.txt
      writing requirements to pyjoulescope_driver.egg-info/requires.txt
      writing top-level names to pyjoulescope_driver.egg-info/top_level.txt
      reading manifest file 'pyjoulescope_driver.egg-info/SOURCES.txt'
      reading manifest template 'MANIFEST.in'
      adding license file 'LICENSE.txt'
      writing manifest file 'pyjoulescope_driver.egg-info/SOURCES.txt'
      /tmp/pip-build-env-qoodm2uj/overlay/lib/python3.9/site-packages/setuptools/command/build_py.py:201: _Warning: Package 'pyjoulescope_driver.test.release' is absent from the `packages` configuration.
      !!
      
              ********************************************************************************
              ############################
              # Package would be ignored #
              ############################
              Python recognizes 'pyjoulescope_driver.test.release' as an importable package[^1],
              but it is absent from setuptools' `packages` configuration.
      
              This leads to an ambiguous overall configuration. If you want to distribute this
              package, please make sure that 'pyjoulescope_driver.test.release' is explicitly added
              to the `packages` configuration field.
      
              Alternatively, you can also rely on setuptools' discovery methods
              (for example by using `find_namespace_packages(...)`/`find_namespace:`
              instead of `find_packages(...)`/`find:`).
      
              You can read more about "package discovery" on setuptools documentation page:
      
              - https://setuptools.pypa.io/en/latest/userguide/package_discovery.html
      
              If you don't want 'pyjoulescope_driver.test.release' to be distributed and are
              already explicitly excluding 'pyjoulescope_driver.test.release' via
              `find_namespace_packages(...)/find_namespace` or `find_packages(...)/find`,
              you can try to use `exclude_package_data`, or `include-package-data=False` in
              combination with a more fine grained `package-data` configuration.
      
              You can read more about "package data files" on setuptools documentation page:
      
              - https://setuptools.pypa.io/en/latest/userguide/datafiles.html
      
      
              [^1]: For Python, any directory (with suitable naming) can be imported,
                    even if it does not contain any `.py` files.
                    On the other hand, currently there is no concept of package data
                    directory, all directories are treated like packages.
              ********************************************************************************
      
      !!
        check.warn(importable)
      /tmp/pip-build-env-qoodm2uj/overlay/lib/python3.9/site-packages/setuptools/command/build_py.py:201: _Warning: Package 'pyjoulescope_driver.test.release.ctrl_app' is absent from the `packages` configuration.
      !!
      
              ********************************************************************************
              ############################
              # Package would be ignored #
              ############################
              Python recognizes 'pyjoulescope_driver.test.release.ctrl_app' as an importable package[^1],
              but it is absent from setuptools' `packages` configuration.
      
              This leads to an ambiguous overall configuration. If you want to distribute this
              package, please make sure that 'pyjoulescope_driver.test.release.ctrl_app' is explicitly added
              to the `packages` configuration field.
      
              Alternatively, you can also rely on setuptools' discovery methods
              (for example by using `find_namespace_packages(...)`/`find_namespace:`
              instead of `find_packages(...)`/`find:`).
      
              You can read more about "package discovery" on setuptools documentation page:
      
              - https://setuptools.pypa.io/en/latest/userguide/package_discovery.html
      
              If you don't want 'pyjoulescope_driver.test.release.ctrl_app' to be distributed and are
              already explicitly excluding 'pyjoulescope_driver.test.release.ctrl_app' via
              `find_namespace_packages(...)/find_namespace` or `find_packages(...)/find`,
              you can try to use `exclude_package_data`, or `include-package-data=False` in
              combination with a more fine grained `package-data` configuration.
      
              You can read more about "package data files" on setuptools documentation page:
      
              - https://setuptools.pypa.io/en/latest/userguide/datafiles.html
      
      
              [^1]: For Python, any directory (with suitable naming) can be imported,
                    even if it does not contain any `.py` files.
                    On the other hand, currently there is no concept of package data
                    directory, all directories are treated like packages.
              ********************************************************************************
      
      !!
        check.warn(importable)
      /tmp/pip-build-env-qoodm2uj/overlay/lib/python3.9/site-packages/setuptools/command/build_py.py:201: _Warning: Package 'pyjoulescope_driver.test.release.ctrl_updater1' is absent from the `packages` configuration.
      !!
      
              ********************************************************************************
              ############################
              # Package would be ignored #
              ############################
              Python recognizes 'pyjoulescope_driver.test.release.ctrl_updater1' as an importable package[^1],
              but it is absent from setuptools' `packages` configuration.
      
              This leads to an ambiguous overall configuration. If you want to distribute this
              package, please make sure that 'pyjoulescope_driver.test.release.ctrl_updater1' is explicitly added
              to the `packages` configuration field.
      
              Alternatively, you can also rely on setuptools' discovery methods
              (for example by using `find_namespace_packages(...)`/`find_namespace:`
              instead of `find_packages(...)`/`find:`).
      
              You can read more about "package discovery" on setuptools documentation page:
      
              - https://setuptools.pypa.io/en/latest/userguide/package_discovery.html
      
              If you don't want 'pyjoulescope_driver.test.release.ctrl_updater1' to be distributed and are
              already explicitly excluding 'pyjoulescope_driver.test.release.ctrl_updater1' via
              `find_namespace_packages(...)/find_namespace` or `find_packages(...)/find`,
              you can try to use `exclude_package_data`, or `include-package-data=False` in
              combination with a more fine grained `package-data` configuration.
      
              You can read more about "package data files" on setuptools documentation page:
      
              - https://setuptools.pypa.io/en/latest/userguide/datafiles.html
      
      
              [^1]: For Python, any directory (with suitable naming) can be imported,
                    even if it does not contain any `.py` files.
                    On the other hand, currently there is no concept of package data
                    directory, all directories are treated like packages.
              ********************************************************************************
      
      !!
        check.warn(importable)
      /tmp/pip-build-env-qoodm2uj/overlay/lib/python3.9/site-packages/setuptools/command/build_py.py:201: _Warning: Package 'pyjoulescope_driver.test.release.ctrl_updater2' is absent from the `packages` configuration.
      !!
      
              ********************************************************************************
              ############################
              # Package would be ignored #
              ############################
              Python recognizes 'pyjoulescope_driver.test.release.ctrl_updater2' as an importable package[^1],
              but it is absent from setuptools' `packages` configuration.
      
              This leads to an ambiguous overall configuration. If you want to distribute this
              package, please make sure that 'pyjoulescope_driver.test.release.ctrl_updater2' is explicitly added
              to the `packages` configuration field.
      
              Alternatively, you can also rely on setuptools' discovery methods
              (for example by using `find_namespace_packages(...)`/`find_namespace:`
              instead of `find_packages(...)`/`find:`).
      
              You can read more about "package discovery" on setuptools documentation page:
      
              - https://setuptools.pypa.io/en/latest/userguide/package_discovery.html
      
              If you don't want 'pyjoulescope_driver.test.release.ctrl_updater2' to be distributed and are
              already explicitly excluding 'pyjoulescope_driver.test.release.ctrl_updater2' via
              `find_namespace_packages(...)/find_namespace` or `find_packages(...)/find`,
              you can try to use `exclude_package_data`, or `include-package-data=False` in
              combination with a more fine grained `package-data` configuration.
      
              You can read more about "package data files" on setuptools documentation page:
      
              - https://setuptools.pypa.io/en/latest/userguide/datafiles.html
      
      
              [^1]: For Python, any directory (with suitable naming) can be imported,
                    even if it does not contain any `.py` files.
                    On the other hand, currently there is no concept of package data
                    directory, all directories are treated like packages.
              ********************************************************************************
      
      !!
        check.warn(importable)
      /tmp/pip-build-env-qoodm2uj/overlay/lib/python3.9/site-packages/setuptools/command/build_py.py:201: _Warning: Package 'pyjoulescope_driver.test.release.sensor_fpga' is absent from the `packages` configuration.
      !!
      
              ********************************************************************************
              ############################
              # Package would be ignored #
              ############################
              Python recognizes 'pyjoulescope_driver.test.release.sensor_fpga' as an importable package[^1],
              but it is absent from setuptools' `packages` configuration.
      
              This leads to an ambiguous overall configuration. If you want to distribute this
              package, please make sure that 'pyjoulescope_driver.test.release.sensor_fpga' is explicitly added
              to the `packages` configuration field.
      
              Alternatively, you can also rely on setuptools' discovery methods
              (for example by using `find_namespace_packages(...)`/`find_namespace:`
              instead of `find_packages(...)`/`find:`).
      
              You can read more about "package discovery" on setuptools documentation page:
      
              - https://setuptools.pypa.io/en/latest/userguide/package_discovery.html
      
              If you don't want 'pyjoulescope_driver.test.release.sensor_fpga' to be distributed and are
              already explicitly excluding 'pyjoulescope_driver.test.release.sensor_fpga' via
              `find_namespace_packages(...)/find_namespace` or `find_packages(...)/find`,
              you can try to use `exclude_package_data`, or `include-package-data=False` in
              combination with a more fine grained `package-data` configuration.
      
              You can read more about "package data files" on setuptools documentation page:
      
              - https://setuptools.pypa.io/en/latest/userguide/datafiles.html
      
      
              [^1]: For Python, any directory (with suitable naming) can be imported,
                    even if it does not contain any `.py` files.
                    On the other hand, currently there is no concept of package data
                    directory, all directories are treated like packages.
              ********************************************************************************
      
      !!
        check.warn(importable)
      copying pyjoulescope_driver/binding.c -> build/lib.linux-armv7l-cpython-39/pyjoulescope_driver
      copying pyjoulescope_driver/img_alpha.img -> build/lib.linux-armv7l-cpython-39/pyjoulescope_driver
      copying pyjoulescope_driver/img_beta.img -> build/lib.linux-armv7l-cpython-39/pyjoulescope_driver
      copying pyjoulescope_driver/img_stable.img -> build/lib.linux-armv7l-cpython-39/pyjoulescope_driver
      creating build/lib.linux-armv7l-cpython-39/pyjoulescope_driver/test/release
      copying pyjoulescope_driver/test/release/index.html -> build/lib.linux-armv7l-cpython-39/pyjoulescope_driver/test/release
      copying pyjoulescope_driver/test/release/index.json -> build/lib.linux-armv7l-cpython-39/pyjoulescope_driver/test/release
      creating build/lib.linux-armv7l-cpython-39/pyjoulescope_driver/test/release/ctrl_app
      copying pyjoulescope_driver/test/release/ctrl_app/index.html -> build/lib.linux-armv7l-cpython-39/pyjoulescope_driver/test/release/ctrl_app
      copying pyjoulescope_driver/test/release/ctrl_app/index.json -> build/lib.linux-armv7l-cpython-39/pyjoulescope_driver/test/release/ctrl_app
      copying pyjoulescope_driver/test/release/ctrl_app/js220_ctrl_app_1_0_1.img -> build/lib.linux-armv7l-cpython-39/pyjoulescope_driver/test/release/ctrl_app
      copying pyjoulescope_driver/test/release/ctrl_app/js220_ctrl_app_1_0_1_CHANGELOG.md -> build/lib.linux-armv7l-cpython-39/pyjoulescope_driver/test/release/ctrl_app
      creating build/lib.linux-armv7l-cpython-39/pyjoulescope_driver/test/release/ctrl_updater1
      copying pyjoulescope_driver/test/release/ctrl_updater1/index.html -> build/lib.linux-armv7l-cpython-39/pyjoulescope_driver/test/release/ctrl_updater1
      copying pyjoulescope_driver/test/release/ctrl_updater1/index.json -> build/lib.linux-armv7l-cpython-39/pyjoulescope_driver/test/release/ctrl_updater1
      copying pyjoulescope_driver/test/release/ctrl_updater1/js220_ctrl_updater1_1_0_1.img -> build/lib.linux-armv7l-cpython-39/pyjoulescope_driver/test/release/ctrl_updater1
      copying pyjoulescope_driver/test/release/ctrl_updater1/js220_ctrl_updater1_1_0_1_CHANGELOG.md -> build/lib.linux-armv7l-cpython-39/pyjoulescope_driver/test/release/ctrl_updater1
      creating build/lib.linux-armv7l-cpython-39/pyjoulescope_driver/test/release/ctrl_updater2
      copying pyjoulescope_driver/test/release/ctrl_updater2/index.html -> build/lib.linux-armv7l-cpython-39/pyjoulescope_driver/test/release/ctrl_updater2
      copying pyjoulescope_driver/test/release/ctrl_updater2/index.json -> build/lib.linux-armv7l-cpython-39/pyjoulescope_driver/test/release/ctrl_updater2
      copying pyjoulescope_driver/test/release/ctrl_updater2/js220_ctrl_updater2_1_0_1.img -> build/lib.linux-armv7l-cpython-39/pyjoulescope_driver/test/release/ctrl_updater2
      copying pyjoulescope_driver/test/release/ctrl_updater2/js220_ctrl_updater2_1_0_1_CHANGELOG.md -> build/lib.linux-armv7l-cpython-39/pyjoulescope_driver/test/release/ctrl_updater2
      creating build/lib.linux-armv7l-cpython-39/pyjoulescope_driver/test/release/sensor_fpga
      copying pyjoulescope_driver/test/release/sensor_fpga/index.html -> build/lib.linux-armv7l-cpython-39/pyjoulescope_driver/test/release/sensor_fpga
      copying pyjoulescope_driver/test/release/sensor_fpga/index.json -> build/lib.linux-armv7l-cpython-39/pyjoulescope_driver/test/release/sensor_fpga
      copying pyjoulescope_driver/test/release/sensor_fpga/js220_fpga_1_0_1.img -> build/lib.linux-armv7l-cpython-39/pyjoulescope_driver/test/release/sensor_fpga
      copying pyjoulescope_driver/test/release/sensor_fpga/js220_fpga_1_0_1_CHANGELOG.md -> build/lib.linux-armv7l-cpython-39/pyjoulescope_driver/test/release/sensor_fpga
      running build_ext
      building 'pyjoulescope_driver.binding' extension
      creating build/temp.linux-armv7l-cpython-39
      creating build/temp.linux-armv7l-cpython-39/pyjoulescope_driver
      creating build/temp.linux-armv7l-cpython-39/src
      creating build/temp.linux-armv7l-cpython-39/src/backend
      creating build/temp.linux-armv7l-cpython-39/src/backend/libusb
      creating build/temp.linux-armv7l-cpython-39/third-party
      creating build/temp.linux-armv7l-cpython-39/third-party/libusb
      creating build/temp.linux-armv7l-cpython-39/third-party/libusb/libusb
      creating build/temp.linux-armv7l-cpython-39/third-party/libusb/libusb/os
      creating build/temp.linux-armv7l-cpython-39/third-party/tinyprintf
      arm-linux-gnueabihf-gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O2 -Wall -g -ffile-prefix-map=/python3.9-3.9.2=. -fstack-protector-strong -Wformat -Werror=format-security -g -fwrapv -O2 -fPIC -I/tmp/pip-install-2826c3f1/pyjoulescope-driver_63df809e92c843fe82c1b9bcfeb50ea2/include -I/tmp/pip-install-2826c3f1/pyjoulescope-driver_63df809e92c843fe82c1b9bcfeb50ea2/include_private -I/tmp/pip-install-2826c3f1/pyjoulescope-driver_63df809e92c843fe82c1b9bcfeb50ea2/third-party/tinyprintf -I/tmp/pip-build-env-qoodm2uj/overlay/lib/python3.9/site-packages/numpy/core/include -I/tmp/pip-install-2826c3f1/pyjoulescope-driver_63df809e92c843fe82c1b9bcfeb50ea2/third-party/libusb/libusb -I/tmp/pip-install-2826c3f1/pyjoulescope-driver_63df809e92c843fe82c1b9bcfeb50ea2/third-party/libusb/include/linux -I/tmp/.venv/include -I/usr/include/python3.9 -c pyjoulescope_driver/binding.c -o build/temp.linux-armv7l-cpython-39/pyjoulescope_driver/binding.o
      In file included from /tmp/pip-build-env-qoodm2uj/overlay/lib/python3.9/site-packages/numpy/core/include/numpy/ndarraytypes.h:1940,
                       from /tmp/pip-build-env-qoodm2uj/overlay/lib/python3.9/site-packages/numpy/core/include/numpy/ndarrayobject.h:12,
                       from /tmp/pip-build-env-qoodm2uj/overlay/lib/python3.9/site-packages/numpy/core/include/numpy/arrayobject.h:5,
                       from pyjoulescope_driver/binding.c:827:
      /tmp/pip-build-env-qoodm2uj/overlay/lib/python3.9/site-packages/numpy/core/include/numpy/npy_1_7_deprecated_api.h:17:2: warning: #warning "Using deprecated NumPy API, disable it with " "#define NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION" [-Wcpp]
         17 | #warning "Using deprecated NumPy API, disable it with " \
            |  ^~~~~~~
      arm-linux-gnueabihf-gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O2 -Wall -g -ffile-prefix-map=/python3.9-3.9.2=. -fstack-protector-strong -Wformat -Werror=format-security -g -fwrapv -O2 -fPIC -I/tmp/pip-install-2826c3f1/pyjoulescope-driver_63df809e92c843fe82c1b9bcfeb50ea2/include -I/tmp/pip-install-2826c3f1/pyjoulescope-driver_63df809e92c843fe82c1b9bcfeb50ea2/include_private -I/tmp/pip-install-2826c3f1/pyjoulescope-driver_63df809e92c843fe82c1b9bcfeb50ea2/third-party/tinyprintf -I/tmp/pip-build-env-qoodm2uj/overlay/lib/python3.9/site-packages/numpy/core/include -I/tmp/pip-install-2826c3f1/pyjoulescope-driver_63df809e92c843fe82c1b9bcfeb50ea2/third-party/libusb/libusb -I/tmp/pip-install-2826c3f1/pyjoulescope-driver_63df809e92c843fe82c1b9bcfeb50ea2/third-party/libusb/include/linux -I/tmp/.venv/include -I/usr/include/python3.9 -c src/backend/libusb/backend.c -o build/temp.linux-armv7l-cpython-39/src/backend/libusb/backend.o
      arm-linux-gnueabihf-gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O2 -Wall -g -ffile-prefix-map=/python3.9-3.9.2=. -fstack-protector-strong -Wformat -Werror=format-security -g -fwrapv -O2 -fPIC -I/tmp/pip-install-2826c3f1/pyjoulescope-driver_63df809e92c843fe82c1b9bcfeb50ea2/include -I/tmp/pip-install-2826c3f1/pyjoulescope-driver_63df809e92c843fe82c1b9bcfeb50ea2/include_private -I/tmp/pip-install-2826c3f1/pyjoulescope-driver_63df809e92c843fe82c1b9bcfeb50ea2/third-party/tinyprintf -I/tmp/pip-build-env-qoodm2uj/overlay/lib/python3.9/site-packages/numpy/core/include -I/tmp/pip-install-2826c3f1/pyjoulescope-driver_63df809e92c843fe82c1b9bcfeb50ea2/third-party/libusb/libusb -I/tmp/pip-install-2826c3f1/pyjoulescope-driver_63df809e92c843fe82c1b9bcfeb50ea2/third-party/libusb/include/linux -I/tmp/.venv/include -I/usr/include/python3.9 -c src/backend/libusb/msg_queue.c -o build/temp.linux-armv7l-cpython-39/src/backend/libusb/msg_queue.o
      arm-linux-gnueabihf-gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O2 -Wall -g -ffile-prefix-map=/python3.9-3.9.2=. -fstack-protector-strong -Wformat -Werror=format-security -g -fwrapv -O2 -fPIC -I/tmp/pip-install-2826c3f1/pyjoulescope-driver_63df809e92c843fe82c1b9bcfeb50ea2/include -I/tmp/pip-install-2826c3f1/pyjoulescope-driver_63df809e92c843fe82c1b9bcfeb50ea2/include_private -I/tmp/pip-install-2826c3f1/pyjoulescope-driver_63df809e92c843fe82c1b9bcfeb50ea2/third-party/tinyprintf -I/tmp/pip-build-env-qoodm2uj/overlay/lib/python3.9/site-packages/numpy/core/include -I/tmp/pip-install-2826c3f1/pyjoulescope-driver_63df809e92c843fe82c1b9bcfeb50ea2/third-party/libusb/libusb -I/tmp/pip-install-2826c3f1/pyjoulescope-driver_63df809e92c843fe82c1b9bcfeb50ea2/third-party/libusb/include/linux -I/tmp/.venv/include -I/usr/include/python3.9 -c src/backend/posix.c -o build/temp.linux-armv7l-cpython-39/src/backend/posix.o
      arm-linux-gnueabihf-gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O2 -Wall -g -ffile-prefix-map=/python3.9-3.9.2=. -fstack-protector-strong -Wformat -Werror=format-security -g -fwrapv -O2 -fPIC -I/tmp/pip-install-2826c3f1/pyjoulescope-driver_63df809e92c843fe82c1b9bcfeb50ea2/include -I/tmp/pip-install-2826c3f1/pyjoulescope-driver_63df809e92c843fe82c1b9bcfeb50ea2/include_private -I/tmp/pip-install-2826c3f1/pyjoulescope-driver_63df809e92c843fe82c1b9bcfeb50ea2/third-party/tinyprintf -I/tmp/pip-build-env-qoodm2uj/overlay/lib/python3.9/site-packages/numpy/core/include -I/tmp/pip-install-2826c3f1/pyjoulescope-driver_63df809e92c843fe82c1b9bcfeb50ea2/third-party/libusb/libusb -I/tmp/pip-install-2826c3f1/pyjoulescope-driver_63df809e92c843fe82c1b9bcfeb50ea2/third-party/libusb/include/linux -I/tmp/.venv/include -I/usr/include/python3.9 -c src/buffer.c -o build/temp.linux-armv7l-cpython-39/src/buffer.o
      arm-linux-gnueabihf-gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O2 -Wall -g -ffile-prefix-map=/python3.9-3.9.2=. -fstack-protector-strong -Wformat -Werror=format-security -g -fwrapv -O2 -fPIC -I/tmp/pip-install-2826c3f1/pyjoulescope-driver_63df809e92c843fe82c1b9bcfeb50ea2/include -I/tmp/pip-install-2826c3f1/pyjoulescope-driver_63df809e92c843fe82c1b9bcfeb50ea2/include_private -I/tmp/pip-install-2826c3f1/pyjoulescope-driver_63df809e92c843fe82c1b9bcfeb50ea2/third-party/tinyprintf -I/tmp/pip-build-env-qoodm2uj/overlay/lib/python3.9/site-packages/numpy/core/include -I/tmp/pip-install-2826c3f1/pyjoulescope-driver_63df809e92c843fe82c1b9bcfeb50ea2/third-party/libusb/libusb -I/tmp/pip-install-2826c3f1/pyjoulescope-driver_63df809e92c843fe82c1b9bcfeb50ea2/third-party/libusb/include/linux -I/tmp/.venv/include -I/usr/include/python3.9 -c src/buffer_signal.c -o build/temp.linux-armv7l-cpython-39/src/buffer_signal.o
      In file included from /tmp/pip-install-2826c3f1/pyjoulescope-driver_63df809e92c843fe82c1b9bcfeb50ea2/include_private/jsdrv_prv/js220_i128.h:24,
                       from src/buffer_signal.c:24:
      /tmp/pip-install-2826c3f1/pyjoulescope-driver_63df809e92c843fe82c1b9bcfeb50ea2/include_private/js220_api.h:315:19: error: ‘__int128’ is not supported on this target
        315 |     __extension__ __int128 i128;
            |                   ^~~~~~~~
      error: command '/usr/bin/arm-linux-gnueabihf-gcc' failed with exit code 1
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for pyjoulescope-driver
Failed to build pyjls pyjoulescope-driver
ERROR: Could not build wheels for pyjls, pyjoulescope-driver, which is required to install pyproject.toml-based projects
```

**Comment by mliberty1** [comment]
Hi @odurc and thanks for reporting this issue.  We must have broke Raspberry Pi support when we added macOS universal2 support in 0.7.1.  We will investigate.

For now, you should be able to install the older packages:
    pip install pyjls==0.7.0 pyjoulescope_driver==1.3.12 joulescope==1.1.6

**Comment by mliberty1** [comment]
Fixed in 0.7.2, which is now available on pypi.  Please reopen if you continue to see issues!
